aspnet/mvc/models/file-uploads.rst

```diff
@@ -0,0 +1,197 @@
+File Uploads
+============
+
+By `Steve Smith`_
+
+ASP.NET MVC actions support uploading of one or more files, using simple model binding for smaller files, or streaming for larger files.
+
+.. contents:: Sections
+  :local:
+  :depth: 1
+
+`View or download sample from GitHub <https://github.com/aspnet/Docs/tree/master/aspnet/mvc/models/file-uploads/sample>`_.
+
+Uploading Small Files with Model Binding
+----------------------------------------
+
+To upload small files, you can use a multi-part HTML form or construct a POST request using JavaScript. An example form using Razor which supports multiple uploaded files is shown below:
+
+.. literalinclude:: file-uploads/sample/FileUploadSample/Views/Home/Index.cshtml
+  :language: html
+  :lines: 7-20
```

Comment:
We prefer named snippets over line numbers. If the code is unlikely to change, you can use line numbers. For razor  targeted to DocFX/OPS you'd use

<!--
<snippet_1>
--> 
    <!--
    </snippet_1>
    -->
[!code-html[Main](start-mvc/sample/src/MvcMovie/Views/Shared/_Layout.html?name=snippet_1&highlight=6,29)]